pocs/observatory.py

```diff
@@ -378,8 +394,9 @@ def update_tracking(self):
                 # Adjust tracking for up to 30 seconds then fail if not done.
                 start_tracking_time = current_time()
                 while self.mount.is_tracking is False:
-                    if (current_time() - start_tracking_time).sec > 30:
-                        raise Exception("Trying to adjust Dec tracking for more than 30 seconds")
+                    if (current_time() - start_tracking_time).sec * 1000 > max_time:
```

Comment:
max_time should have a unit in its name (e.g. _ms), or better yet should be a datetime.timedelta object.